src/Sulu/Bundle/ContentBundle/Admin/ContentContentNavigationProvider.php

```diff
@@ -82,7 +78,7 @@ public function getNavigationItems(array $options = array())
             $permissions->setComponentOptions(
                 array(
                     'display' => 'form',
-                    'type' => Structure::class,
+                    'type' => PageDocument::class,
```

Comment:
I do not require any methods, as said this is just an unique identifier for the object type in the acl. What subclasses does the BasePageDocument have? If it is HomeDocument and PageDocument it would work fine, why shouldn't I use it then?